README.md

```diff
@@ -549,7 +549,7 @@ List of all available attributes can be found at [otel-webserver-module](https:/
 
 #### Using Nginx autoinstrumentation
 
-For `Nginx` autoinstrumentation, Nginx versions 1.22.0, 1.23.0, and 1.23.1 are supported at this time. The Nginx configuration file is expected to be `/etc/nginx/nginx.conf` by default, if it's different, see following example on how to change it. Instrumentation at this time also expects, that `conf.d` directory is present in the directory, where configuration file resides and that there is a `include <config-file-dir-path>/conf.d/*.conf;` directive in the `http { ... }` section of Nginx configuration file (like it is in the default configuration file of Nginx). You can also adjust OpenTelemetry SDK attributes. Example:
+For `Nginx` autoinstrumentation, Nginx versions 1.22.0, 1.23.0, 1.23.1, 1.24.0, 1.25.3, 1.25.5, and 1.26.0 are supported at this time. The Nginx configuration file is expected to be `/etc/nginx/nginx.conf` by default, if it's different, see following example on how to change it. Instrumentation at this time also expects, that `conf.d` directory is present in the directory, where configuration file resides and that there is a `include <config-file-dir-path>/conf.d/*.conf;` directive in the `http { ... }` section of Nginx configuration file (like it is in the default configuration file of Nginx). You can also adjust OpenTelemetry SDK attributes. Example:
```

Comment:
Are you confident all of these versions are actually supported? When we previously bumped this, there was some disagreement over this.

Reply:
I was just basing this off of the release notes. 1.24.0 and 1.25.3 were added in `1.0.4` and 1.26.0 and 1.25.5 were added in `1.1.0`. Support for 1.22.0, 1.23.0, and 1.23.1 correspond with the release notes for version `1.0.2`, and in `1.0.3` no change in supported versions was noted. So it seems like the supported versions weren't added for the `1.0.4` release. But I'm purely basing this off the notes [here](https://github.com/open-telemetry/opentelemetry-cpp-contrib/releases). I haven't actually deployed or tested anything yet, I just noted the outdated version and thought I'd help update it as it seemed simple enough.

Reply:
This is the instrumentation we test the least, so I'm always a bit more cautious when the version is bumped. For example, if you look at the [README](https://github.com/aryanishan1001/opentelemetry-cpp-contrib/blob/main/instrumentation/otel-webserver-module/README.md#platform-supported-1), it claims to only support 1.24 and 1.25.

On the other hand, it looks like this was already the case for `1.0.4`, which we blindly upgraded to without any complaints, so maybe this instrumentation simply isn't used that much, or its users carefully set the image tag in their CRs.

@pavolloffay @jaronoff97 @frzifus WDYT? Do we just bite the bullet and upgrade?

Reply:
@swiatekm I just checked the created container and it the versions `1.25.5` and `1.26.0` in it for NGINX.

Reply:
Can we extend the test for some period of time to just check a few versions at the same time? wdyt?